### PR TITLE
Implement new simplecast player.

### PR DESCRIPTION
**Pull request by jpetto**
Demo:

http://sandbox.equalparts.io/irlpodcast/

### CODE REVIEW

**Comment by jpetto** [comment]
Note that the embed IDs for episodes 0-4 were purposefully updated. (The old IDs were redirecting to the new IDs for the legacy player only, and don't work in the new player.)

File: _sass/irlpodcast.scss [hunk 1]
```diff
@@ -441,13 +399,15 @@ ol {
     }
 }
 
+// episode title is included in the embedded player, so no need to actually display
+// kept in markup for a11y
 .episode-title {
-    @include font-size-level3();
+    @include visually-hidden();
```
Comment (stephaniehobson) [inline _sass/irlpodcast.scss]: 👍

**Comment by jpetto** [comment]
Updated the home page layout to *mostly* match the episode page. Also added a grey box behind player while it loads for some visual reference (Vince's suggestion).

**Comment by stephaniehobson** [comment]
R+ 🏃

**Pull request merged**